ydb/public/lib/ydb_cli/commands/ydb_service_scheme.cpp

```diff
@@ -543,6 +550,50 @@ int TCommandDescribe::DescribeReplication(const TDriver& driver) {
     return PrintDescription(this, OutputFormat, result, &TCommandDescribe::PrintReplicationResponsePretty);
 }
 
+int TCommandDescribe::PrintViewResponsePretty(const NKikimrSchemeOp::TViewDescription& desc) const {
+    Cout << Endl << "Query text: " << desc.GetQueryText() << Endl;
+    return EXIT_SUCCESS;
+}
+
+int TCommandDescribe::DescribeView(TConfig& config) {
+    const NKikimr::NGRpcProxy::TGrpcError* transportError = nullptr;
+    NKikimrClient::TResponse response;
+    {
+        auto clientConfig = NYdbGrpc::TGRpcClientConfig(config.Address);
+        if (config.EnableSsl) {
+            clientConfig.EnableSsl = config.EnableSsl;
+            clientConfig.SslCredentials.pem_root_certs = config.CaCerts;
```

Comment:
I have tested:
- no authentication
- YDB token authentication

I imagine I would need to test all of the other methods to be sure that the GRPC client is configured correctly 😢